### PR TITLE
Add banquet placement list

### PR DESCRIPTION
**Pull request by apals**
screen-banq

### CODE REVIEW

File: app/src/main/java/se/ths/kth/Aramda/ui/SearchBarBehavior.java [hunk 1]
```diff
@@ -28,11 +29,13 @@ public SearchBarBehavior(Context context, AttributeSet attrs) {
 
     @Override
     public boolean onStartNestedScroll(CoordinatorLayout coordinatorLayout, FrameLayout child, View directTargetChild, View target, int nestedScrollAxes) {
+        Log.d(TAG, "onStartNestedScroll: aa");
```
Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/ui/SearchBarBehavior.java]: pls remove

File: app/src/main/java/se/ths/kth/Aramda/ui/SearchBarBehavior.java [hunk 2]
```diff
@@ -28,11 +29,13 @@ public SearchBarBehavior(Context context, AttributeSet attrs) {
...
         return (nestedScrollAxes & ViewCompat.SCROLL_AXIS_VERTICAL) != 0;
     }
 
     @Override
     public void onNestedScroll(CoordinatorLayout coordinatorLayout, FrameLayout child, View target, int dxConsumed, int dyConsumed, int dxUnconsumed, int dyUnconsumed) {
+        Log.d(TAG, "onNestedScroll: bb");
```
Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/ui/SearchBarBehavior.java]: delete

**Comment by Jeppes** [comment]
feel free to merge once the logging statements have been removed

**Comment by apals** [comment]
fixed

**Comment by apals** [comment]
screen-banq-dialog

**Pull request merged**